modules/kubernetes/eks-core/k8s-role-binding.tf

```diff
@@ -139,3 +139,24 @@ resource "kubernetes_role_binding" "sa_helm_release" {
     namespace = kubernetes_namespace.service_accounts.metadata[0].name
   }
 }
+resource "kubernetes_role_binding" "top" {
+  for_each = { for ns in var.namespaces : ns.name => ns }
+  metadata {
+    name      = "${each.value.name}-top"
+    namespace = kubernetes_namespace.tenant[each.key].metadata[0].name
+    labels = {
+      "aad-group-name"    = var.aad_groups.edit[each.key].name
+      "xkf.xenit.io/kind" = "platform"
+    }
+  }
+  role_ref {
+    api_group = "rbac.authorization.k8s.io"
+    kind      = "ClusterRole"
+    name      = "top"
```

Comment:
Use this way of getting the name instead:
`kubernetes_cluster_role.top.metadata[0].name`

Reply:
Fixed